python/lsst/ts/hexrotcomm/base_csc.py

```diff
@@ -65,7 +73,7 @@ def make_connect_error_info(prefix, connected, connect_descr):
 
 # TODO DM-39787: remove this function once MTHexapod supports
 # MTRotator's simplified states.
-def make_state_transition_dict():
+def make_state_transition_dict() -> dict:
```

Comment:
If possible, please specify the dict contents. Feel free to leave this for a future ticket.

Reply:
Added in the commit: [4f2c704](https://github.com/lsst-ts/ts_hexrotcomm/pull/69/commits/4f2c704e3a773371b0ab61a92cfae6b9acd21c33). Thanks!